Processes/Docs-Contribution-Process-External.md

```diff
@@ -0,0 +1,44 @@
+# Docs Contribution Process - External
+
+## Overview
+This document provides instructions for contributors external to the ENF Foundation on how to contribute new or updated content to [EOS Documentation](https://docs.eosnetwork.com).
+
+### When to Use this Procedure
+Use this procedure when you need to update content from one or repositories to  [EOS Documentation](https://docs.eosnetwork.com).
+
+### Expected Results
+Fresh, updated content is available for consumption in EOS Documentation.
+
+## How to Contribute Content to EOS 
+1. Fork the docs repository
+2. Create a new branch
+3. Create/edit/rename files
+4. Create a Pull Request with the target as this repository's main branch
+5. Request a review
+
+# Overview
+
+The ENF Developer Relations team (DevRel) has developed a set of templates to assist development of documentation for the EOS ecosystem. The templates are written in Markdown (.md) format and provide a structure for your content contribution. Take advantage of these writing tools to assist with your content contribution. These templates are guidelines for various types of content and will help you organize your content with a common look and feel. These templates are not a substitute for content accuracy and completeness.
+
+You can provide content to the DevRel team in various formats. You can rely on the DevRel team to perform a thorough edit and review of your content before it is published to the EOS Documentation portal. The publication process includes  uses GitHub workflows to publish Markdown (.md) formatted content.
```

Comment:
"The publication process includes  uses GitHub workflows to publish Markdown (.md) formatted content."

This sentence doesn't make sense

Reply:
Fixed - "The publication process uses GitHub workflows to publish Markdown (.md) formatted content."

Done